content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +15,190 @@ further_reading:
+
+1. Hover over a recorded test step and click the **Set Cursor** icon. A blue line appears above your test step. 
+2. Record additional [test steps](#automatically-recorded-steps) or add [steps manually](#manually-added-steps).
+3. When you are done adding additional steps above your tests step, click **Clear Cursor** to exit.
+
+{{< img src="synthetics/browser_tests/recording_cursor_step.mp4" alt="Set the cursor on a test step to add additional steps before this step" video="true" width="100%" >}}
 
 ## Automatically recorded steps
 
-The following steps are automatically recorded with the [Datadog browser test recorder extension][3]:
+Once you click **Start Recording**, the [Datadog browser test recorder extension][3] automatically detects and records steps on your website.
 
 ### Click
 
-[Datadog browser test recorder extension][3] automatically records clicks on your page. Specify the type of click you want your browser test to perform at execution time:
+Interacting with elements on your page records a step on the left hand corner. 
 
-{{< img src="synthetics/browser_tests/browser_test_click_step.mp4" alt="Browser Test Click Step" video="true" width="60%">}}
+{{< img src="synthetics/browser_tests/click_step.mp4" alt="Click type dropdown menu in the Click step type" video="true" width="60%" >}}
 
-Choose from:
+Click on the step and select a click type you want the browser test to perform at execution time:
 
-* Primary click (corresponds to a left click)
+* Primary click corresponding to a left click
 * Double click
-* Contextual click (corresponds to a right click)
+* Contextual click corresponding to a right click
 
 ### Type text
 
-[Datadog browser test recorder extension][3] automatically records text input in any fields of your website (such as forms or text areas):
+Typing or inputting text into fields on your website such as forms or text areas records a step on the left hand corner.
 
-{{< img src="synthetics/browser_tests/input_text.mp4" alt="Browser Test Input Text Step" video="true" width="100%">}}
+{{< img src="synthetics/browser_tests/input_text.mp4" alt="Browser Test Input Text Step" video="true" width="95%" >}}
 
 ### Select option
 
-[Datadog browser test recorder extension][3] automatically records options being selected from `select` dropdown menu:
+Selecting an option from a `select` dropdown menu records a step on the left hand corner.
 
-{{< img src="synthetics/browser_tests/select_options.png" alt="Select options step" style="width:60%;">}}
+{{< img src="synthetics/browser_tests/select_options.png" alt="Select options step" style="width:70%;" >}}
 
 ### Upload file
 
-You can record the uploading of files as a step. To record an **Upload** step you can:
+Uploading a file records a step on the left hand corner. You can upload up to 10 files with a limit of 5MB each.
```

Comment:
Not a big fan to say we record on the left hand corner.
Steps you perform on your apps are recorded in DD and we show the recap of the recording on the left hand corner. but saying we record it in the left hand corner feels a bit weird to me